postgresql/postgresql_client.go

```diff
@@ -0,0 +1,167 @@
+package postgresql
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/internal/config"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/internal/iam"
+	pg "github.com/Azure/azure-sdk-for-go/services/preview/postgresql/mgmt/2020-02-14-preview/postgresqlflexibleservers"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+// GetServersClient returns
+func GetServersClient() pg.ServersClient {
+	serversClient := pg.NewServersClient(config.SubscriptionID())
+	a, _ := iam.GetResourceManagementAuthorizer()
+	serversClient.Authorizer = a
+	serversClient.AddToUserAgent(config.UserAgent())
+	return serversClient
+}
+
+// CreateServer creates a new PostgreSQL Server
+func CreateServer(ctx context.Context, serversClient pg.ServersClient, serverName string, dbLogin string, dbPassword string) (server pg.Server, err error) {

```

Comment:
Also we could merge those parameters that have same type, like this:
```suggestion
func CreateServer(ctx context.Context, serversClient pg.ServersClient, serverName, dbLogin, dbPassword string) (server pg.Server, err error) {

```

Reply:
fixed